spec/superspawn.spec.js

```diff
@@ -31,9 +30,12 @@ describe('spawn method', function () {
         failSpy = jasmine.createSpy('fail'); /* eslint no-unused-vars : 0 */
     });
 
-    it('Test 001 : should return a promise', function () {
-        expect(Q.isPromise(superspawn.spawn(LS))).toBe(true);
-        expect(Q.isPromise(superspawn.spawn('invalid_command'))).toBe(true);
+    it('should resolve on success', () => {
+        return expectAsync(superspawn.spawn(LS)).toBeResolved();
```

Comment:
Possibly off-topic: I think it should be possible to use a very dumb command such as `echo` that is exactly the same between Windows cmd, ps, and *nix *sh (including Windows "*nix*" *sh). This should probably be done in a separate PR, someday. Not a blocker.

Reply:
Would be nice. Although I hope we will sunset `superspawn` before that :wink: